### PR TITLE
Internal Reports: Add eager loading based on Bullet

### PR DESCRIPTION
**Pull request by jacobherrington**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

Bullet suggested doing some eager loading on these tables. Makes sense to me. :slightly_smiling_face: 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

external-content duckduckgo com

### CODE REVIEW

File: app/controllers/internal/feedback_messages_controller.rb [hunk 1]
```diff
@@ -3,7 +3,9 @@ class Internal::FeedbackMessagesController < Internal::ApplicationController
 
   def index
     @q = FeedbackMessage.
-      includes(:reporter, :notes).
+      includes(:reporter,
```
Comment (rhymes) [inline app/controllers/internal/feedback_messages_controller.rb]: why the three lines :D ?

Comment (jacobherrington) [reply app/controllers/internal/feedback_messages_controller.rb]: 3 > 1

Comment (jacobherrington) [reply app/controllers/internal/feedback_messages_controller.rb]: To be honest, this is just how emacs thought it should look and I didn't want to argue. :grimacing:

Comment (rhymes) [reply app/controllers/internal/feedback_messages_controller.rb]: lol

**Pull request merged**